gabbi/httpclient.py

```diff
@@ -10,7 +10,6 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-"""Subclass of Http class for verbosity."""
```

Comment:
This seemed wrong/outdated?

Reply:
It is outdated, it was a subclass of httplib2's Http class. Now it is is a subclass of urllib3's PoolManager operating as an HTTP client (urllib3 does this layering thing where methods are cascaded up through various classes by a combination of composition and inheritance.